include/Track.h

```diff
@@ -133,6 +133,43 @@ class LMMS_EXPORT TrackContentObject : public Model, public JournallingObject
 	{
 		return m_autoResize;
 	}
+	
+	unsigned int color() const
+	{
+		return( m_color.rgb() );
+	}
+	
+	QColor colorObj() const
+	{
+		return m_color;
+	}
+
+	void setColor( const QColor & c )
+	{
+		m_color = QColor( c );
+	}
+	
+	void setColorRgb( const unsigned int & c )
+	{
+		m_color.setRgb( c );
+	}
```

Comment:
```suggestion
	unsigned int colorRgb() const
	{
		return( m_color.rgb() );
	}
	
	QColor color() const
	{
		return m_color;
	}

	void setColor( const QColor & c )
	{
		m_color = QColor( c );
	}
	
	void setColorRgb( const unsigned int & c )
	{
		m_color.setRgb( c );
	}
```

More consistent naming?

Reply:
Sure! Was a copy paste from BB color implementation, guess that needs a change.

Reply:
Done.